### DISCUSSION TITLE
Organization and Co-pilot

### DISCUSSION THREAD

**Discussion by DarklinuxFr** [community/community#67808] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello, I received an email, saying that I was entitled to copilot chat, but I read the instructions and strangely, on my github/organization pane, I cannot find the copilot logo, so is that I don't understand the instructions, or does my github account not have access to this functionality? THANKS

**Comment by Akash1134** [comment, marked as answer]
Hi @DarklinuxFr ,

If you've received an email for Copilot Chat entitlement, please check whether it is for your Individual or Business account. We recently announced Copilot Chat public beta for Individual users. 

If you want to setup Copilot Chat on your Organization, please refer [Using Copilot Chat](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat).

Thanks!
